### ISSUE TITLE
Ensure aria-roledescription is only used on elements with an implicit or explicit role (Accessibility Insights)

### ISSUE THREAD

**Issue by Tsarar** [microsoft/BotFramework-WebChat#3688]
## Screenshots
![image](https://user-images.githubusercontent.com/24234334/106130695-5aa32100-616a-11eb-95d0-5c8574b14e8a.png)
![image](https://user-images.githubusercontent.com/24234334/106130722-61319880-616a-11eb-972f-d8c42bd0fbcb.png)

## Version
npm 4.9.2-master.8696a8e

<meta name="botframework-directlinespeech:version" content="4.9.2">
<meta name="botframework-webchat:bundle:variant" content="full-es5">
<meta name="botframework-webchat:bundle:version" content="4.9.2">
<meta name="botframework-webchat:core:version" content="4.9.2">
<meta name="botframework-webchat:ui:version" content="4.9.2">
<meta name="botframework-webchat-adapter-ic3" content="version=0.1.0-master.d105296">

## Describe the bug

Accessibility check throws an error

## Steps to reproduce

Sent a message
Run accessibility check with https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni

## Expected behavior

Accessibility check should not throw any errors

**Comment by Tsarar** [comment]
Please, remove Bot Servies and customer-reported labels and add Area: Accessibility and

**Comment by srinaath** [comment]
@amal-khalaf Could you take a look at this ticket?

**Comment by corinagum** [comment]
Please test this behavior on the latest version of Web Chat (4.11.0). Behavior should be resolved in latest release.

**Comment by amal-khalaf** [comment]
this is fixed in webchat 4.11 
closing the bug as not repro.
